ballerina/natives.bal

```diff
@@ -103,7 +103,7 @@ public function lookupMetric(string name, map<string>? tags = ()) returns Counte
 # + name - Name of the counter metric
 # + description - Description of the counter metric
 # + metricTags - Tags associated with the counter metric
-public  class Counter {
+public class Counter {
```

Comment:
Can we make this an isolated object too?

Reply:
Yes we can.